samples/sample_fei/src/fei_encode.cpp

```diff
@@ -739,7 +739,7 @@ mfxStatus FEI_EncodeInterface::EncodeOneFrame(iTask* eTask)
             {
                 // ignore warnings if output is available
                 sts = m_pmfxSession->SyncOperation(m_SyncPoint, MSDK_WAIT_INTERVAL);
-                MSDK_CHECK_STATUS(sts, "FEI ENCODE: SyncOperation failed");
+                MSDK_CHECK_ERR_NONE_STATUS(sts, MFX_ERR_ABORTED, "SyncOperation failed");
```

Comment:
I liked previous message, when failed interface explicitly mentioned it is easier to read error messages. I know that this information still can be extracted from code line, but it adds a little bit more work, especially when you code base and error log mismatches